playground/package.json

```diff
@@ -9,8 +9,8 @@
     "postinstall": "nuxi prepare"
   },
   "dependencies": {
-    "@trpc/client": "^10.44.1",
-    "@trpc/server": "^10.44.1",
+    "@trpc/client": "^11.0.0-next-beta.190",
+    "@trpc/server": "^11.0.0-next-beta.190",
```

Comment:
There is a slight change in the `infer`-types since we've removed the "RootConfig" from the resolved procedures' types